src/WarcraftLegacies.Source/Mechanics/Scourge/HelmOfDominationDropsWhenScourgeLeaves.cs

```diff
@@ -1,22 +1,25 @@
-// using WarcraftLegacies.Source.Legends;
-// using WarcraftLegacies.Source.Setup;
-// using WarcraftLegacies.Source.Setup.FactionSetup;
-//
-// using static War3Api.Common;  using static MacroTools.GeneralHelpers;
-// {
-//   public class HelmOfDominationDropsWhenScourgeLeaves{
-//
-//     private static void Actions( ){
-//       if (GetTriggerFaction() == ScourgeSetup.FactionScourge && ArtifactSetup.ArtifactHelmofdomination.OwningUnit == LegendScourge.LegendLichking.Unit){
-//         SetItemPosition(ArtifactSetup.ArtifactHelmofdomination.Item, GetRectCenterX(gg_rct_LichKing), GetRectCenterY(gg_rct_LichKing));
-//       }
-//     }
-//
-//     public static void Setup( ){
-//       trigger trig = CreateTrigger();
-//       OnFactionGameLeave.register(trig);
-//       TriggerAddAction(trig,  Actions);
-//     }
-//
-//   }
-// }
+using WarcraftLegacies.Source.Setup.Legends;
+using WarcraftLegacies.Source.Setup;
+using WarcraftLegacies.Source.Setup.FactionSetup;
+using static War3Api.Common;
+using MacroTools.FactionSystem;
+using MacroTools.Extensions;
+
+public class HelmOfDominationDropsWhenScourgeLeaves
+{
+    public static void Setup()
```

Comment:
Could we add a quick documentation document here? E.g.:
```cs
/// <summary>
///  When Scourge leaves the game, drop the Helm of Domination to an easily accessible position since the default position is hard to see.
/// </summary>
```